builtin/settingtypes.txt

```diff
@@ -796,6 +796,13 @@ autoscale_mode (Autoscaling mode) enum disable disable,enable,force
 #    Show entity selection boxes
 show_entity_selectionbox (Show entity selection boxes) bool true
 
+#    If 'true', the skybox displays horizon haze, sunrise/set horizon glow and
+#    terrain horizons that hide the sun, moon and stars.
+#    Set to 'false' to create the appearence of space (if time is set to night)
+#    or a 'terrainless' air world (if time is set to day).
+#    If 'false', 'directional colored fog' is not effective so is disabled.
+skybox_far_terrain (Show skybox far terrain) bool true
```

Comment:
This should be a hud flag, really, as it's a game play related thing

Reply:
2nd commit removed.